planet4-gutenberg-blocks.php

```diff
@@ -135,6 +136,7 @@
 	'planet4-blocks/split-two-columns',
 	'planet4-blocks/submenu',
 	'planet4-blocks/timeline',
+	'planet4-blocks/socialshare',
```

Comment:
The list is alphabetized. Can you please move this below `planet4-blocks/social-media`?